fe/fe-core/src/main/java/com/starrocks/server/GlobalStateMgr.java

```diff
@@ -1656,6 +1656,8 @@ public void loadImage(String imageDir) throws IOException, DdlException {
             checksum = loadAnalyze(dis, checksum);
             remoteChecksum = dis.readLong();
             checksum = loadWorkGroups(dis, checksum);
+            checksum = auth.readAsGson(dis, checksum);
```

Comment:
Does it work when you load an old version image?

Reply:
I suggest that you'd better to implement all the persistent in another PR. It seems that your change will not be compatible with old version meta image.

Reply:
I suggest that you'd better to implement all the persistent in another PR. It seems that your change will not be compatible with old version meta image.

Reply:
If FE upgrade then downgrade, the old FE won't read this part of the image. I have also test it. The only problem is that I add an operation type in edit log, but user can use a `ignore_unknown_log_id` to skip it.